docs/connector-development/config-based/tutorial/0-getting-started.md

```diff
@@ -42,6 +42,7 @@ This can be done by signing up for the Free tier plan on [Exchange Rates Data AP
 
 - An Exchange Rates API key
 - Python >= 3.9
+- [Poetry](https://python-poetry.org/)
 - Docker must be running
 - NodeJS
```

Comment:
We don't really require Node — that's a remnant from the time of generator running without docker. That's a nitpick — I know it's there and needs to be cleaned, just never got to it ;(